README.md

```diff
@@ -41,17 +53,9 @@ These issues are marked below as "requires manual review".
   - *Why CoinGecko? CoinGecko's token list updates every hour which means we get token list updates very quickly. CoinGecko also uses an in-depth [listing criteria](https://www.coingecko.com/en/methodology).*
 
 #### Debugging Automated checks failures
-If your automated checks failed, you can see the reason for the failure by downloading `validation-artifacts.zip`, unzipping it and opening the `validation_results.txt` file. To locate the `validation-artifacts` follow these steps:
 
-1. Click on the `Details` link for `Validate PR` check in your PR
-2. Click `Summary` in the left panel
-3. Find the section on the page labeled `Artifacts` and click on `validation-artifacts`
-4. After download of `validation-artifacts.zip`, unzip it and open `validation_results.txt`
+To run the automated checks run `pnpm check`. Pnpm check happens automatically when you open your project in stackblitz if using stackblitz to create your PR.
 
-If you make changes and need to run the validation check again, you will need to wait for a reviewer to approve the checks to run again. However, if you do not want to wait for a reviewer to approve the checks to run again to see if the failures have been resolved, you can run the validation checks locally by running:
-```
-npx tsx ./bin/cli.ts validate --datadir ./data --tokens <data folder name (e.g. ETH)>
-```
```

Comment:
It hink that these instructions are still valuable for people that do not want to use stackblitz, so it might be good to leave them in.

Reply:
ok that's fair

Reply:
i think they work! i've been using them for the github actions, lol couldn't find them at first (so at least keep the one about finding the artifacts)